Project-Lifecycle.md

```diff
@@ -98,6 +98,9 @@ vote to abstain from representation on the TSC.
 
 ## Applying to join
 
+**SPECIAL NOTICE:**
+Project [admittance is on hold](#admittance).
```

Comment:
Personally I'd rather not add this as it could end up having a significant chilling effect even down the road if/when we decided to open it up. Also, it may detract from the possibility of bringing in other small projects under core that do not warrant their own TLP.